### PR TITLE
Encryption

### PR DESCRIPTION
**Pull request by SandeepVasireddy**
Encrypted passwords and created sessions for users

### CODE REVIEW

**Comment by sophtank** [comment]
Current setup to create dummy data will not work if the program is run more than once, as it creates dummy data upon every instantiation, which will cause a unique key violation error. What I am proposing in the groupchat is that you create dummy accounts via the log in manually, and add those dummy accounts into the insert statements for "users" in "final_project_schema.sql". So create user "svasire4" manually in your browser, grab the hashed password that bcrypt puts into the SQL, and then add that to final_project_schema. I recommend that you keep the usernames as our NinerNet usernames so the dummy data created in other insert statements still works.

File: requirements.txt [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
+Flask-Bcrypt==1.0.1
+iniconfig==2.0.0
 itsdangerous==2.1.2
 Jinja2==3.1.3
 MarkupSafe==2.1.5
-psycopg==3.1.18
-psycopg-pool==3.2.1
-python-dotenv==1.0.1
-typing_extensions==4.10.0
-tzdata==2024.1
```
Comment (Zzz4nder) [inline requirements.txt:13]: Is there a reason to delete this? Was it a dependency of another packet?

File: final_project_schema.sql [hunk 2]
```diff
@@ -6,7 +6,7 @@ DROP TABLE IF EXISTS users; --run the rest of the code now
 
 CREATE TABLE IF NOT EXISTS users (
     username VARCHAR(50),
-    password VARCHAR(50) NOT NULL, --may need to change the length once we decide hashing length, for now store in plaintext
+    password VARCHAR(255) NOT NULL, --may need to change the length once we decide hashing length, for now store in plaintext
```
Comment (Zzz4nder) [inline final_project_schema.sql:9]: delete comment?

**Review by Zzz4nder — APPROVED** [review]
Left some comments on the code for clarification, but it lgtm

File: final_project_schema.sql (hunk 2 above)
Comment (SandeepVasireddy) [reply final_project_schema.sql:9]: ill delete the comment

**Pull request merged**